synchro/views/landing_pages/controllers.py

```diff
@@ -335,6 +337,9 @@ def landing_page(page, version, prod_category):
     if variant_default:
       template_vars['is_variant'] = noindex
 
+  if page == 'gold':
+    return redirect(shopify_page_base + page + '-' + version, code=301)
```

Comment:
For the base `/gold/` page, this will add an extra hyphen in the redirect url: `/gold-/`. Will push an update soon.